Scripts/Terminology-Install-Utilities.psm1

```diff
@@ -355,6 +408,12 @@ function Get-TerminologyConfig {
     }
     $sqlLogDirectoryConfig = Get-ConfigValue -Prompt "Log directory to create Terminology database" -DefaultFromParam $SqlLogDirectory -DefaultFromInstallConfig $sqlLogDirectoryParam -Quiet:$Quiet
 
+    # Loader User
+    $loaderWindowsUser = Get-ConfigValue -Prompt "Loader windows username" -AdditionalPromptInfo "(ex. DOMAIN\USER)" -DefaultFromParam $LoaderWindowsUserParam -DefaultFromInstallConfig $installSettings.loaderWindowsUser -Quiet:$Quiet
```

Comment:
Small thing but "windows" should probably be capitalized

Reply:
👍